### ISSUE TITLE
Fallout 4 not starting after new DLC (dlcworkshop3.esm)and patch was installed

### ISSUE THREAD

**Issue by lordzallen** [TanninOne/modorganizer#901]
Fallout 4 crashes when the initial video sequence is going.

I did a trace with visual studio debugger and got this error

Unhandled exception at 0x00007FFFB5FEC6D5 (msvcr120.dll) in Fallout4.exe: 0xC0000005: Access violation reading location 0x0000018DABD1F000.

Exception thrown at 0x00007FFFB5FEC6D5 (msvcr120.dll) in Fallout4.exe: 0xC0000005: Access violation reading location 0x0000018DABD1F000.

hope this information helps.

fallout 4 version 1.6.9.0

system specs amd fx3820e OC'ed 4.5 ghz stable
asrock 990fx extreme 3 flashed to most recent bios
sapphire r9 380 4gb

if this is duplicate, please merge my information with the appropriate issue tracker.

**Comment by lordzallen** [comment]
disabled the intro video. fallout 4 still crashes a second or two in.

**Comment by lordzallen** [comment]
[usvfs-2016-07-26--7-27.zip](https://github.com/TanninOne/modorganizer/files/386498/usvfs-2016-07-26--7-27.zip)

my logs

**Comment by lordzallen** [comment]
windows 10 pro x64 for os

**Comment by GSDFan** [comment]
If you have DEF Ui installed hide HUDMenu.swf in the interface folder and see if that helps.

**Comment by Grestorn** [comment]
That's not caused by MO. Many users are affected and this is caused by certain mods. Try to disable all .esm/esp files until the game starts to figure out which mod is the cause. Also look for updates to all mods.

**Comment by lordzallen** [comment]
i disabled all my mods still crashes a second or two in.

**Comment by BetaVirus** [comment]
same issue. i disabled all mod and reinstalled all tools with updated or current versions (FO4EDIT, LOOT, F4SE) but still crashing before intro.

**Comment by Gernash** [comment]
Are you running MO2 in portable mode?
Have you a clean install of your FO4 folder (i.e redownload and patch) if you were using NMM

**Comment by BetaVirus** [comment]
I had to do a clean install of FO4 2 weeks ago.

Before i installed mods using NMM, I was watching videos about merging mod and making custom mods; Several times MO was mentioned.

So after research and installing the MO2, I found MO2 did not actually modify you data folder. So corruptions, and no loose files hiding about.

I validated my install today and all is good. No reinstall as I just did that 2 weeks ago. I was playing only a few hours before the Steam update. Then logged out and now MO2 will not load the game or LOOT. Bother updated to current. 

*Running LOOT with out MO2 works fine but it does not see the mods I have.
**Disabled ALL mods in MO2 and even made a new profile but still crashing with all mods disabled.

(I have only been playing FO4 for 2 weeks with MO2 and only managed to make it to Sanctuary. So much research on how to use MO2 and how it sees mod files and how organization of the mod works vs the old NMM day. Why would anyone not use MO?!)

I will installed a new copy of MO2 tomorrow in a new directory and see if that helps. Mod conversion is easy if it does.

**Comment by azraal** [comment]
I have the same issue.
So it's clearly not related to the fallout install since I can launch it without issue without MO
- Created a new empty profile and it's still fail
- Created a new empty profile with save and config separated
- Launched each one without f4se or with it
- Cannot even launch the fallout launcher from MO
- For the Loot exit code, I have 3221225477 which correspond to 0xC0000005 (access violation) which is also consistant with xEdit error.

For the record Windows 10 x64 pro (french version, but it doesn't seem relevant here)

**Comment by Gernash** [comment]
Running Loot from the programs folder you should see fallout 4 and the DLC's no mods.
if after you clean install of FO4 you installed mods with NMM you may have "dirtied" the installation (residue files). You may have to reinstall your FO4.

@azraal I have all DLC,Loot F4se everything running no issues. that;s why I think it may be either FO4 install issue or Permissions in MO2 profile/Mod folder.

If you decide to try reinstalling FO4 then running it once from Steam then install MO2 then try Loot if working then throw a couple of mods. There is no reason Loot should not be working that is where my focus would be. Something is causing the issue and it would be what is stopping you.

**Comment by Gernash** [comment]
have you tries this? [http://forum.step-project.com/topic/11038-installing-mo2-for-fallout-4-beta-issues/](http://forum.step-project.com/topic/11038-installing-mo2-for-fallout-4-beta-issues/)

**Comment by BetaVirus** [comment]
For my Win10 Pro x64, all AV and antispyware installed by or part of MS is disabled, i do not have any other software installed.

I may not have been clear when I said i have a clean install of FO4. I have NOT used NMM since the reinstall and converting to MO2. 
*I have NMM installed but only to keep track of updates for the mod that i use. Also, for some reason the MO2 will not let me download the mods directly to/from MO2, so i have to manually have MO2 import the mod after downloading through NMM.   Annoying but the MO2 option to associate the nxm links does not do anything.

**Comment by Gernash** [comment]
Nxm links workaround are cover in the howto I linked you. Run the nxmhandler.exe in the MO2 folder. 
I assume it's a genuine copy running 1.6.9?
Do you have a 3rd party AV software?
If you open ini file editor in MO2 is fallout4.ini and fallout4prefs.ini there? Do they have a heap of settings?

**Comment by BetaVirus** [comment]
I do have a plethora of settings in my ini.
 This issue only happened after the new DLC. Nothing changed but that. Not sure how new DLC cause MO2 to start crashing.
Let me know if you need logs or anything.

Sent from Yahoo Mail on Android 
 
  On Thu, Jul 28, 2016 at 8:17, Gernash<notifications@github.com> wrote:   
Nxm links workaround are cover in the howto I linked you. Run the nxmhandler.exe in the MO2 folder. 
I assume it's a genuine copy running 1.6.9?
Do you have a 3rd party AV software?
If you open ini file editor in MO2 is fallout4.ini and fallout4prefs.ini there? Do they have a heap of settings?

**Comment by SnowbankNL** [comment]
I had all the same problems. For me it ended being the achievements mod (that enables achievements on modded games trough F4SE) that crashes me. I manually installed that F4SE plugin and everything is working for me now.

**Comment by BetaVirus** [comment]
So after hours of hack and slash testing, it turns out the update broke the Looksmenu mod and any mod that uses it as a base.

for me it was Simply Gorgy Look for looksmenu and looks menu. Oddly even when i disabled all it still crashed, only by disabling one at a time did it work

**Comment by lordzallen** [comment]
still not working after disabling mods.

**Comment by Gernash** [comment]
Running 3rd party mods like achievements unlocker that have dll's in the fo4 folder?

**Comment by lordzallen** [comment]
just skse i think

**Comment by BetaVirus** [comment]
Agreed, F4SE always breaks when new updates happens, I just never had issues with a mod as long as the F4SE was updated and working.


I will remember to keep eye on F4SE based mods.

**Comment by azraal** [comment]
Ok for those who are blocked by f4se mods (those with dll) if you can't do without them (save broken or something), you'll have to install them in your game directory.
It's a bit against the purpose of modorganizer, but it will do as a workaround...

**Comment by lordzallen** [comment]
apparently there was some stuff mucking up my fallout 4. reinstalled and loaded. works. had to do bodyslide workarounds tho
